easybuild/easyconfigs/a/APBS/APBS-3.0.0_Linux.eb

```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# TH77EMBL update do 3.0.0, 2021/04
+name = 'APBS'
+version = '3.0.0'
+versionsuffix = '_Linux'
```

Comment:
Maybe tag this as `x86-64`? Linux is kind of a given...

But +1 on building this from source instead...